### ISSUE TITLE
Space in the name of the main file

### ISSUE THREAD

**Issue by leorolla** [vlmantova/bookml#19]
If the name of the main file contains a space, BookML 0.9.1 will ignore it.

How to reproduce:
Rename the main file
Run "make"

Tested on Ubuntu 24.04

**Comment by xworld21** [comment]
Hard! GNU make really, really does not like spaces in file names.

It is theoretically possible to support spaces in filenames, somehow. I am using certain GNU features that make it even more difficult (second expansion!) so it's going to take some time and testing before a fix, assuming I can make it work at all.

**Comment by leorolla** [comment]
Indeed! Maybe a full fix is not worth the cost, but this issue can still be added to the documentation, and perhaps "make" can issue a warning message in case any file with space is present.
